content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -48,24 +70,36 @@ once.
 
 <admon icon="book">
 
-`dvc exp run` is an experiment-specific alternative to `dvc repro`.
-`dvc exp save` can be used to capture experiments after executing ML processes
-manually.
+`dvc exp run` is an experiment-specific alternative to `dvc repro`. See [Running
+Pipelines] for differences between them.
 
 </admon>
 
 [reproduction targets]: /doc/command-reference/repro#options
 [dependency graph]: /doc/user-guide/pipelines/defining-pipelines
+[running pipelines]: /doc/user-guide/pipelines/running-pipelines
+
+### Running the pipeline(s) on cloud
+
+You can use [Studio] to run the experiment pipeline on a remote compute instance
```

Comment:
Minor: Add some screenshots?

Reply:
Making it a follow-up because I think it needs to be a bit more stable for me to capture good screen shots.